src/api/app/models/staging/staging_project.rb

```diff
@@ -11,7 +11,7 @@ class StagingProject < Project
     before_create :add_managers_group
 
     def staging_identifier
-      name[/.*:Staging:(.*)/, 1]
+      name[/.*:Staging:(.*)/, 1] || name
```

Comment:
Why do we need this at all?

Reply:
The identifier?

Reply:
I would argue this should be default. But this is nitpicking...

Reply:
The staging_identifier already does that. If you meant to overwrite the `name`... I wouldn't do that since this might create side effects. For example when storing the staging project we also store the `name`.

Reply:
We just discussed this in person and concluded that we don't want to have a staging identifier. Since there is no hard rule to identify what the identifier would be.

Reply:
I find it fine like that. Otherwise it will return `nil` if it doesn't include `Staging` :confused:

Reply:
We dropped the staging identifier now.

Reply:
:angry: